test/e2e/runner/rpc.go

```diff
@@ -55,7 +55,7 @@ func waitForHeight(testnet *e2e.Testnet, height int64) (*types.Block, *types.Blo
 		if len(clients) == 0 {
 			return nil, nil, errors.New("unable to connect to any network nodes")
 		}
-		if time.Since(lastIncrease) >= 20*time.Second {
+		if time.Since(lastIncrease) >= time.Minute {
```

Comment:
I think increasing this timeout doesn't matter since the reason that this was timing out was because it takes a little longer to start up but you've increased the startup timeout (start.go) from 15 seconds to a minute anyway.

Reply:
I believe that this is used both for startup and for catch up after perturbations, so its useful to change.